blink/README.md

```diff
@@ -2,19 +2,29 @@
 
 ## Overview
 
-The [Blink][1] integration is a collection of no-code actions and triggers usable in Blink Automations. Instead of writing manual API calls and scripts, you can use Blink's visual editor to drag-and-drop Datadog API actions into your workflow or use one of the hundreds of existing pre-made automations from the [library][2].
+[Blink][1] is a low-code/no-code (LCNC) platform that enables automated incident response, cloud-native operations, and security operations workflows. Blink transforms manual tasks into interactive automations backed by the security and reliability of a cloud-native platform. Every script or ticket becomes a fully-managed automation.
 
-This integration enables you to:
-1. Trigger automations within Blink in response to new Datadog incidents 
-2. Create and update Datadog incidents in Blink 
-3. View incidents or events from the Datadog event stream in Blink
+The user interface and [automation library][2] come with premade Datadog-based automations and use-cases. Blink helps you achieve better cloud efficiency and more competitive SLA's, with fewer operational bottlenecks.
 
-View our full list of actions and capabilities on the [documentation site][3].
+This out-of-the-box integration enables you to:
+
+- Trigger event-based Blink automations using Datadog incidents.
+- Create and update Datadog incidents automatically from within Blink.
+- View incidents or events from the Datadog Events Explorer in Blink.
+- Automatically enrich and remediate Datadog incidents using Blink automations.
+
+For more information about Blink, see the [Blink documentation][3].
 
 ## Setup
 
 Visit [our documentation][4] for details on how to connect your Datadog workspace to Blink.
```

Comment:
Hi @liavt, apologies for the delay. Due to a recent incident, we were in a code freeze for the last two weeks. Could you confirm this document will be updated with instructions specific to OAuth?

Reply:
@eho1307 Yes, once the Oauth is deployed we will update the documentation link with instructions on how to connect